open-metadata-implementation/user-interfaces/presentation-server/cra-server/index.js

```diff
@@ -0,0 +1,70 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+const express = require('express');
+const fs = require("fs");
+const path = require("path")
+const session = require("express-session");
+const bodyParser = require("body-parser");
+let passport = require("passport");
+const https = require("https");
+const app = express();
+require("dotenv").config();
+
+const getServerInfoFromEnv = require('./functions/getServerInfoFromEnv');
+const serverNameMiddleWare = require('./functions/serverNameMiddleware');
+const passportConfiguration = require('./functions/passportConfiguration');
+const loggedIn = require('./functions/loggedIn');
+
+const router = require('./router/routes');
+
+const PORT = process.env.PORT || 8091;
+const env = process.env.NODE_ENV || 'development';
+
+
+// ssl self signed certificate and key
+const cert = fs.readFileSync(path.join(__dirname, '../') + "/../presentation-server/ssl/keys/server.cert");
```

Comment:
this is probably something we'll need to return to later (so we have consolidated certs/clear docs)- but should go with this for now

Reply:
agreed

Reply:
let's handle this in #1403

Reply:
#1403